app/assets/javascripts/application.js

```diff
@@ -6,11 +6,15 @@
 //= require length_counter
 //= require markdown_preview
 //= require toggle_display_with_checked_input
-
+//= require paste-html-to-govspeak
 jQuery(function ($) {
   $('.js-length-counter').each(function () {
     new GOVUK.LengthCounter({ $el: $(this) }) // eslint-disable-line no-new
   })
 
+  $('.js-paste-html-to-govspeak').each(function () {
+    this.addEventListener('paste', pasteHtmlToGovspeak.pasteListener) // eslint-disable-line no-undef
```

Comment:
rather than using `// eslint-disable-line no-undef` it's more conventional to declare this as a global. (example: https://github.com/alphagov/govuk_publishing_components/blob/379868b6208626d8b2fd1a35f18537cdbc58e592/app/assets/javascripts/govuk_publishing_components/components/accordion.js#L1) 

So instead put a comment at the top of the file of `/* global pasteHtmlToGovspeak */`